crates/biome_html_formatter/tests/specs/html/component-frameworks/astro-expressions-whitespace.astro.snap

```diff
@@ -0,0 +1,57 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: component-frameworks/astro-expressions-whitespace.astro
+---
+# Input
+
+```astro
+<div>
+	<p>I'm just <span>testing</span> something</p>
+	<p>{name} ({username})</p>
+	<p>Hello {firstName} {lastName}!</p>
+	<p>{a} + {b} = {sum}</p>
+</div>
+
+```
+
+
+=============================
+
+# Outputs
+
+## Output 1
+
+-----
+Indent style: Tab
+Indent width: 2
+Line ending: LF
+Line width: 80
+Attribute Position: Auto
+Bracket same line: false
+Whitespace sensitivity: css
+Indent script and style: false
+Self close void elements: never
+-----
+
+```astro
+<div>
+	<p>
+		I'm just <span>testing</span> something
+	</p>
+	<p>{name}({username})</p>
+	<p>Hello {firstName} {lastName}!</p>
+	<p>{a}+ {b}= {sum}</p>
```

Comment:
1. The space between `{name}` and `({username})` disappeared.
1. The spaces before the `+` and `=` disappeared.